src/hotspot/share/oops/method.cpp

```diff
@@ -401,6 +401,7 @@ void Method::metaspace_pointers_do(MetaspaceClosure* it) {
   NOT_PRODUCT(it->push(&_name);)
 }
 
+#if INCLUDE_CDS
```

Comment:
Do you need to ifdef out the declaration in method.hpp?

Reply:
Fixed. I also found out that `Method::restore_unshareable_info` also needed to be ifdef'ed out.

Reply:
Updates look good.